dandi/consts.py

```diff
@@ -75,7 +75,7 @@
 
 #: Regular expression for a valid Dandiset version identifier.  This regex is
 #: not anchored.
-VERSION_REGEX = fr"(?:{PUBLISHED_VERSION_REGEX}|draft)"
+VERSION_REGEX = rf"(?:{PUBLISHED_VERSION_REGEX}|draft)"
```

Comment:
curious: is there a difference and/or standardization of the order of `r` and `f`?

Reply:
There's no practical difference, but the latest version of black prefers `rf` for some reason.

Reply:
gotcha -- thank you!